code/src/index.css

```diff
@@ -5,9 +5,157 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #181818;
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+* {
+  margin: 0;
+  padding: 0;
+}
+/* ===== HEADER ===== */
+header {
+  margin: 20px 23px;
+  border-bottom: 2px solid #252525;
+}
+
+h1 {
+  font-size: 20px;
+  margin: 10px auto;
+}
+
+/* ===== CARDS ===== */
+.App {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.song-card {
+  margin: 10px;
+  width: 300px;
+  display: flex;
+  flex-direction: column;
+}
+
+.song-info {
+  margin: 10px 0;
+}
+
+.song-name {
+  margin-bottom: 5px;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.song-artist {
+  font-size: 12px;
+  color: #a8a8a8;
+}
+
+/* Display structure for when there are multiple artists */
+.song-artist:not(:first-of-type, :last-of-type)::before {
+  content: ", ";
+}
+
+.song-artist:nth-last-of-type(2)::after {
+  content: " & ";
+}
+
+.song-artist a {
+  text-decoration: none;
+}
+
+.song-name a:visited,
+.song-name a:link {
+  color: white;
+  text-decoration: none;
+}
+
+.img-container {
+  position: relative;
+}
+
+.cover {
+  opacity: 1;
+  transition: 0.5s ease-in;
+  z-index: 1;
+}
+
+.overlay {
+  position: absolute;
+  width: 300px;
+  height: 300px;
+  top: 0;
+}
+
+.icons {
+  position: absolute;
+  height: 300px;
+  width: 300px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  transition: 0.5s ease;
+  cursor: pointer;
+  opacity: 0;
+}
+
+/* ===== HOVER ===== */
+
+.img-container:hover .cover {
+  opacity: 0.3;
+}
+
+.img-container:hover .icons {
+  opacity: 1;
+}
+
+.play-icon:hover {
+  transform: scale(1.2);
+  transition: 0.4s ease;
+}
+
+.song-info a:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+/* Tablet */
+
+@media (min-width: 667px) {
```

Comment:
We're not quite sure why but if we look at the page in the inspector tool, in console, it says: Failed to load resource: the server responded with a status of 404 ()
But the page looks good in tablet still :)